### DISCUSSION TITLE
Ability to Clone/Duplicate a Report

### DISCUSSION THREAD

**Discussion by kenderson4** [cortezaproject/corteza-server#1747] [Ideas]
Hey Team! There are many times where we start from a report and want to evolve it, or create a variant from it. In which case it would be amazing to have a "Clone" or "Duplicate" option in the dropdown shown in the attached image. Feel free to let me know if there any any questions related to this. Thank you!
![Screenshot 2024-03-29 144302](https://github.com/cortezaproject/corteza/assets/142103508/57027fff-66dc-4175-b159-c4cf1ae819cc)

**Comment by Fajfa** [comment]
Created an issue, closing discussion
https://github.com/cortezaproject/corteza/issues/1750
